install-manager/src/test/java/cm/aptoide/pt/install_manager/TestData.kt

```diff
@@ -394,7 +399,7 @@ internal class TaskInfoRepositoryMock : TaskInfoRepository {
 
   private var allCalled = false
   private val saveCalledFor: MutableSet<TaskInfo> = mutableSetOf()
-  private val removeAllCalledFor: MutableSet<String> = mutableSetOf()
+  private val removeAllCalledFor: MutableSet<Long> = mutableSetOf()
```

Comment:
Since removal is now not by a primitive but as a whole object this should be also based on the whole object rather than on `Long` value.
So let's fix this mock to test without repository implementation details assumptions.

Reply:
Fixed